raiden/ui/cli.py

```diff
@@ -68,6 +68,13 @@
         type=str,
     ),
     click.option(
+        '--rpccorsdomain',
+        help='Comma separated list of domains to accept cross origin requests. \n'
+        '(localhost enabled by default)',
+        default=None,
```

Comment:
They are definitely two different domains, but that doesn't mean they are obligatory values that should always be CORS enabled.